lib/addons/p5.dom.js

```diff
@@ -1837,6 +1837,70 @@
    * @param {Number}            [val] volume between 0.0 and 1.0
    * @return {Number|p5.MediaElement} current volume or p5.MediaElement
    * @method volume
+   * @example
+	 * <div>
+	 * <code>
+	 * var audio;
+	 * var colorLow;
+	 * var colorMid;
+	 * var colorHigh;
+	 * var counter = 0;
+	 * 
+	 * function loaded() {
+	 *   audio.play();
+	 * }
+	 * 
+	 * function low() {
+	 *   audio.volume(0.1);
+	 * }
+	 * 
+	 * function mid() {
+	 *   audio.volume(0.5);
+	 * }
+	 * 
+	 * function high() {
+	 *   audio.volume(0.9);
+	 * }
+	 * 
+	 * function setup() {
+	 *   colorLow = color(255, 0, 0);
+	 *   colorMid = color(255, 255, 0);
+	 *   colorHigh = color(0, 255, 0);
+	 *   audio = createAudio('assets/lucky_dragons_-_power_melody.mp3', loaded);
+	 *   textAlign(CENTER);
+	 * }
+	 * 
+	 * function draw() {
+	 *   if (counter === 0) {
+	 *     background(colorHigh);
+	 *     text('volume(0.9)', width / 2, height / 2);
+	 *   } else if (counter === 1) {
+	 *     background(colorMid);
+	 *     text('volume(0.5)', width / 2, height / 2);
+	 *   } else if (counter === 2) {
+	 *     background(colorLow);
+	 *     text('volume(0.1)', width / 2, height / 2);
+	 *   }
+	 * }
+	 * 
+	 * function mouseClicked() {
+	 *   if (mouseX > 0 && mouseX < width && mouseY < height && mouseY > 0) {
+	 *     counter++;
+	 *     if (counter === 0) {
+	 *       high();
```

Comment:
rather than defining the functions above, you could just put the single lines of code right here in the if statements. this will simplify the example a bit.